index.bs

```diff
@@ -168,8 +168,8 @@ Note: Sanitizing a string will use the [=HTML Parser=] to parse the input,
 
 ## The Configuration Dictionary {#config}
 
-The <dfn lt="configuration">sanitizer's configuration object</dfn> is a
-dictionary which describes modifications to the sanitze operation.
+The sanitizer's <dfn>configuration object</dfn> is a
+dictionary which describes modifications to the sanitize operation.
```

Comment:
I'm not sure if this is an editorial comment, or a comment on how the defaults work.

Editorial: The config certainly does describe a modification to the sanitize operation. 

Defaults: The way I've specified it here is that the default is a proper default, and a user supplied config replaces the default. (As opposed to the baseline, which is always enforced.) I've now changed it to be more granular, so that the default allow lists is used whenever no allow list is supplied. That is, a user config with a block list would apply the user's block list + the default allow list. Is this what you meant?